README.rst

```diff
@@ -40,21 +40,32 @@ In order to install the package in the current environment:
 
     maturin develop
 
-Build and Release:
+Release
+=======
+
+Update version in ``Cargo.toml`` and:
 
 .. code-block ::
 
     vim Cargo.toml
     git ci -am "Bump version"
     git tag -a v1.2.3
+    git push v1.2.3
 
-Update version in ``Cargo.toml`` and:
+Publish wheel for your host OS:
 
 .. code-block ::
 
     maturin build
     maturin publish
 
+
+Publish wheels of all architectures on PyPi:
+
+1. Download artifacts from Github Actions run on tag v1.2.3. On the bottom of the `Publish wheels` workflow summary page, download the `pypi_files.zip` and extract it locally.
```

Comment:
> run on tag v1.2.3

I'm guessing `v1.2.3` is an example value? It kind of tripped me up when I read it -- maybe we can make that clear?